src/v2/Apps/Gene/geneRoutes.tsx

```diff
@@ -15,12 +15,22 @@ const GeneShowRoute = loadable(() => import("./Routes/GeneShow"), {
 
 export const geneRoutes: RouteConfig[] = [
   {
-    path: "/gene2/:slug",
+    path: "/gene/:slug",
     getComponent: () => GeneApp,
     prepare: () => {
       return GeneApp.preload()
     },
     children: [
+      {
+        path: "artists",
```

Comment:
I probably just don't know how to read this route file offhand — but is there an `:gene/artists` subpath I missed? (Or was there in the old app?)

Reply:
There was when I was tearing down the routes and I figured it was the artists tab in the old page... but apparently it doesn't work and the tab links to a query param. https://www.artsy.net/gene/representations-of-architecture/artists — so it looks like this can be safely deleted, in fact.

Reply:
Gotcha, thanks